### DISCUSSION TITLE
APIman Source code

### DISCUSSION THREAD

**Discussion by t-sharabi** [apiman/apiman#2415] [Q&A] [answered]
Hello,
i was trying the APIman and noticed that you provide the source code on the Github, my question is about this code, is it a total free source code or just the deploying code? in another words, can we freely customize the code that is on the Github?

thanks

**Comment by msavy** [comment, marked as answer]
Apiman's source code is Apache License 2.0, which is an extremely permissive license. Please refer to the following for details: https://snyk.io/learn/apache-license/

**Reply by t-sharabi** [reply]
Hello
Thank you very much for the quick response, I have read the link and I feel that i need to understand the following from you if you can help:
1- can our dev team change any codes that is related to algorithms inside the core code?
2- can our dev team change features or add new feature to the platform?

thanks

**Reply by msavy** [reply]
Yes. 

Caveat: I'm not a lawyer. I'm the co-founder and maintainer of the project.

We welcome contributions back to the project.
